crates/rpc/rpc-types/src/eth/engine/payload.rs

```diff
@@ -220,6 +221,23 @@ pub struct BlobsBundleV1 {
     pub blobs: Vec<Bytes>,
 }
 
+impl From<Vec<BlobTransactionSidecar>> for BlobsBundleV1 {
+    fn from(sidecars: Vec<BlobTransactionSidecar>) -> Self {
+        // TODO: either upstream a conversion or upstream QUANTITY serde impls
+        let (commitments, proofs, blobs) = sidecars.into_iter().fold(
+            (Vec::new(), Vec::new(), Vec::new()),
+            |(mut commitments, mut proofs, mut blobs), sidecar| {
+                commitments
+                    .extend(sidecar.commitments.iter().map(|sidecar| Bytes::from(sidecar.deref())));
+                proofs.extend(sidecar.proofs.iter().map(|proof| Bytes::from(proof.deref())));
+                blobs.extend(sidecar.blobs.iter().map(|blob| Bytes::from(blob.deref())));
+                (commitments, proofs, blobs)
+            },
```

Comment:
yeah this is a bit bad because this effectively clones everything, right?

ideally there's a into_bytes or to_vec impl

Reply:
Ended up using the kzg types now that the serde traits are upstreamed